services/deployment-agent/.env

```diff
@@ -7,4 +7,4 @@ SIMCORE_STACK_NAME=simcore
 # defines the dockerhub registry organisation
 SIMCORE_DOCKER_REGISTRY=itisfoundation
 # set the image tag to be used from dockerhub
-SIMCORE_IMAGE_TAG=master-latest
+SIMCORE_IMAGE_TAG=master-github-latest
```

Comment:
why the ``gitbhu`` reference? As a default to this repo vs the environment repos in gitlab?

Reply:
that are the images generated by github actions